### PR TITLE
SOS-102   add MUI theme support

### PR DESCRIPTION
**Pull request by SeanTConrad**
Added MUI themes. Set theme in .env with REACT_APP_THEME.

Added footer too so you can see the theme in action. Colors and company name are from theme.

### CODE REVIEW

File: client/src/components/index.ts [hunk 1]
```diff
@@ -39,3 +39,4 @@ export { default as PayrollSummary} from './PayrollSummary';
 export { default as BurnChart} from './BurnChart';
 export { default as PLComponent} from './PLComponent';
 export { default as SinglePage} from './SinglePage'
+export { default as Footer} from './Footer'
```
Comment (aylusltd) [inline client/src/components/index.ts:42]: please add a newline at the end of the files

File: client/src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            }}
+          >
+            <Typography variant="caption" color="initial">
+              Copyright ©{(new Date().getFullYear())} {theme.custom.company}
+            </Typography>
+          </Box>
+        </Container>
+
+    );
+  }
```
Comment (aylusltd) [inline client/src/components/Footer.tsx:60]: please add a newline here

**Review by aylusltd — CHANGES_REQUESTED** [review]

File: client/src/components/index.ts (hunk 1 above)
Comment (SeanTConrad) [reply client/src/components/index.ts]: Added to all files in the PR.

File: client/src/components/Footer.tsx (hunk 2 above)
Comment (SeanTConrad) [reply client/src/components/Footer.tsx]: done

**Review by aylusltd — APPROVED** [review]

**Pull request merged**